### DISCUSSION TITLE
configServer 目前进度如何，都有哪些功能了

### DISCUSSION THREAD

**Discussion by wangzan18** [alibaba/ilogtail#1403] [Help] [answered]
我有1000台虚拟机，想使用 configServer 管理，目前可以使用了么？都有哪些功能了

**Comment by Takuka0311** [comment, marked as answer]
目前可以使用，[使用介绍](https://ilogtail.gitbook.io/ilogtail-docs/config-server/quick-start)
最核心的功能是可以让所有机器的心跳发送到ConfigServer，然后通过ConfigServer批量管理这些机器上的采集配置
